libs/ui/src/components/EditAttributeModal/EditAttributeModal.test.tsx

```diff
@@ -245,6 +248,47 @@ describe('EditAttributeModal', () => {
                 }
             });
         });
+
+        test('Submit maxLength field on change', async () => {
+            const user = userEvent.setup();
+            jest.spyOn(gqlTypes, 'useGetAttributeByIdQuery').mockImplementation(
+                () => mockResultGetAttributeById as QueryResult
+            );
+            const mockSaveAttributeMutation = jest.fn().mockReturnValue({
+                data: {
+                    saveAttribute: {
+                        ...mockAttributeWithDetails
+                    }
+                }
+            });
+            jest.spyOn(gqlTypes, 'useSaveAttributeMutation').mockImplementation(() => [
+                mockSaveAttributeMutation,
+                {loading: false, called: false, client: null, reset: null, error: null}
+            ]);
+
+            render(<EditAttributeModal attributeId={mockAttributeWithDetails.id} open onClose={jest.fn()} />);
+
+            const maxLengthField = screen.getByRole('spinbutton', {name: /max_length/i});
+            expect(maxLengthField).not.toBeDisabled();
+
+            const maxLength = 5;
+
+            await user.type(screen.getByRole('spinbutton', {name: 'global.max_length'}), maxLength.toString());
+
+            await act(async () => {
+                fireEvent.focus(screen.getByRole('spinbutton', {name: /max_length/i}));
+                fireEvent.blur(screen.getByRole('spinbutton', {name: /max_length/i}));
+            });
```

Comment:
**nitpick:** Can be replace (https://github.com/testing-library/user-event/issues/189#issuecomment-877062542)
```suggestion
            await userEvent.click(document.body); // move the focus away
```